source/lookup_table.hpp

```diff
@@ -57,17 +51,13 @@ class AddressLookupTable
 		}
 
 		T *pAddress8 = nullptr;
```

Comment:
No need for a temporary variable here: Can just return the result in the branch below.

Reply:
Updated.  Removed temporary variable.